build_helpers/cssRollup.config.js

```diff
@@ -0,0 +1,84 @@
+import multiEntry from '@rollup/plugin-multi-entry';
+import postcss from 'rollup-plugin-postcss';
+import path from 'path';
+import autoprefixer from 'autoprefixer';
+import postcssCustomProperties from 'postcss-custom-properties';
+import cssVar from '../src/stubs/cssVar';
+
+function escapeSlash(match) {
+  return match.replace(/\//g, '_');
+}
+
+function slashTransform(content) {
+  return content.replace(/\.[\w\/\:\.]+(\s|\,)/g, escapeSlash);
+}
+
+// NOTE (pradeep): Our custom rollup plugin that transforms slashes to underscores in CSS files
+// Check https://rollupjs.org/guide/en/#plugins-overview for a quick start on roll plugins
+function cssSlashTransformPlugin() {
+  return {
+    name: 'plugin-css-slash-transform', // this name will show up in warnings and errors
+    resolveId(source) {
+      if (source.match(/\.css$/)) {
+        return source; // this signals that rollup should not ask other plugins or check the file system to find this id
+      }
+      return null; // other ids should be handled as usually
+    },
+    transform(code, id) {
+      if (id.match(/\.css$/)) {
+        return slashTransform(code);
+      }
+      return null; // other ids should be handled as usually
+    },
+  };
+}
+
+// NOTE (pradeep): I prefer CSS to be at "dist/css", but this breaks backward compatibility
+// because previously we always bundled CSS in "dist/"
+const cssOutputDirectory = 'dist/';
```

Comment:
What are folks' thoughts on this?

Earlier, we used to put css in `dist/`. 
We also put UMD build files in the same directory.

While refactoring the build system, I thought about placing css in `dist/css` and umd builds in `dist/umd`.
This looks cleaner to me, but we can't do it without breaking compatibility.
I personally think not breaking compatibility is better than the slight improvements to imports here...

Reply:
Yeah, breaking compatibility is an issue.  Best to leave it